src/opm/input/eclipse/Schedule/Well/Well.cpp

```diff
@@ -533,6 +534,7 @@ Well Well::serializationTestObject()
     result.wvfpexp = std::make_shared<WVFPEXP>(WVFPEXP::serializationTestObject());
     result.m_pavg = PAvg();
     result.well_temperature = 10.0;
+    result.well_inj_mult = std::make_shared<InjMult>();
```

Comment:
I am not familiar regarding this. Can I use the default constructor or I have to use something like 

`std::make_shared<InjMult>(InjMult::serializationTestObject())` here? @bska @akva2

Reply:
default constructed objects do not test that the serialization works. that is the whole point of the serializationTestObject member so you likely want to do the latter. why is it a shared pointer?

Reply:
Good point. I guess it does not need to, since the Well Object is already in a shared_pointer. Will update.